ami/main/api/views.py

```
@@ -155,6 +155,7 @@ class ProjectViewSet(DefaultViewSet, ProjectMixin):
     serializer_class = ProjectSerializer
     pagination_class = ProjectPagination
     permission_classes = [ObjectPermission]
+    ordering_fields = ["name", "created_at", "updated_at"]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)